src/util.js

```diff
@@ -179,5 +179,25 @@ Util.isMobile = function() {
   return check;
 };
 
+// Adapted from https://mdn.io/Object/assign#Polyfill
```

Comment:
it's not natively supported in any version of WebKit and not in IE 11 (but in Edge, yes): http://kangax.github.io/compat-table/es6/#test-Object_static_methods_Object.assign